File: 1302-delete-characters-to-make-fancy-string/1302-delete-characters-to-make-fancy-string.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        ans = s[0]
        cnt = 1
        for i, c in enumerate(s[1:]):
            if c == ans[-1]:
                cnt += 1
                if cnt > 2:
                    continue
                else:
                    ans += c
            else:
                ans += c
                cnt = 1
        return ans

File: 1302-delete-characters-to-make-fancy-string/1302-delete-characters-to-make-fancy-string.cpp
class Solution {
public:
    string makeFancyString(string s) {
        string ans;
        ans += s[0];
        int cnt = 1;
        for(int i = 1; i < s.size(); i++){
            if(s[i] == ans.back()){
                cnt++;
                if(cnt > 2){
                    continue;
                }
                else{
                    ans += s[i];
                }
            }
            else{
                ans += s[i];
                cnt = 1;
            
            }
        }
        return ans;
    }
};